package.yaml

```diff
@@ -63,6 +63,7 @@ dependencies:
   - file-embed == 0.0.*
   - filelock == 0.1.*
   - filepath == 1.4.*
+  - finite-field == 0.10.*
```

Comment:
Do we need to depend on this package as we have a source copy of the required module in our codebase?

Reply:
That module depends on another module from the package which you didn't include. It doesn't compile without it.